_includes/manuals/1.10/4.2.4_smart_vars.md

```diff
@@ -43,6 +47,8 @@ Once you've created the defaults for your smart variable, you can then add at le
     </tr>
 </tbody></table>
 
+![Example](/static/images/screenshots/param_classes_new/smart_variable_matcher_row.png)
```

Comment:
Also this image contains an internal Red Hat domain name, I'd suggest using "example.com" or similar RFC-protected name.